recipes/emi/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "emi" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{version}}
+
+source:
+  # path: ./
+  url: https://github.com/BioPhys-Turin/EfficientMutualInformation/archive/refs/tags/0.1.0.tar.gz 
+  sha256: 492641e36e659d7fc6ec0466649aae85b01dfb8a187606a937e793f2909aa3c8
+
+build:
+  number: 0
+  skip: true  # [win, osx]
+
+requirements:
+  build:
+    - cmake
+    - make
+    - gxx  # Use `clang` on macOS
+
+  host: 
+    - gxx
+    - libgcc
+    - libstdcxx
+  
+  run:
+    - gxx
+    - libgcc
+    - libstdcxx
+
+test: 
+  commands:
+    - $PREFIX/bin/hk
```

It should be in the path, if not we have a problem.
I'm not sure if there is a `--help`, but it would be nice to test this CLI with something.

```suggestion
    - hk --help
```